test-runner-frontend/src/Components/IdCell.js

```diff
@@ -5,8 +5,9 @@ import CheckCircleOutlinedIcon from '@mui/icons-material/CheckCircleOutlined';
 import HighlightOffOutlinedIcon from '@mui/icons-material/HighlightOffOutlined';
 import HelpOutlineIcon from '@mui/icons-material/HelpOutline';
 import HourglassBottomIcon from '@mui/icons-material/HourglassBottom';
+import { Link } from 'react-router-dom';
 
-export default function IdCell({ value, hasCheckbox, onChange, status }) {
+export default function IdCell({ value, hasCheckbox, onChange, status, row }) {
```

Comment:
do we need to pass the whole row object?

Reply:
Yes we need it for the Report view.